pulp_rpm/app/migrations/0017_merge_advisory_collections.py

```diff
@@ -0,0 +1,53 @@
+from django.db import (
+    migrations,
+    models,
+    )
```

Comment:
Weird formatting here?  I think this can go on one line

Reply:
I think it's just last part of parenthesis should be at the beginning of the line

Reply:
yus - Black has taught me "more than one import in a single from goes to multiline" :)